services/operatorsinfo/operatorsinfo_subgraph_test.go

```diff
@@ -0,0 +1,60 @@
+package operatorsinfo
+
+import (
+	"context"
+	"testing"
+
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	"github.com/Layr-Labs/eigensdk-go/types"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/shurcooL/graphql"
+	"github.com/stretchr/testify/assert"
+)
+
+type mockGraphQLQuerier struct {
+	QueryFn func(ctx context.Context, q any, variables map[string]any) error
+}
+
+func (m mockGraphQLQuerier) Query(ctx context.Context, q any, variables map[string]any) error {
+	return m.QueryFn(ctx, q, variables)
+}
+
+func TestIndexedChainState_GetIndexedOperatorState(t *testing.T) {
+	logger := logging.NewNoopLogger()
```

Comment:
don't use noop loggers in tests, I did that before but that's a mistake I realized. Because when you run go tests they are silent by default, and only if you add `-v` then they show output. But if we use a noopLogger then even adding `-v` won't print anything.

Reply:
got it.  Anyways not logging anything here, removed it for now

Reply:
Ok added Zap logger - that should be ok?

Reply:
yep